grafana-plugin/src/plugin/GrafanaPluginRootPage.tsx

```diff
@@ -153,7 +156,7 @@ export const Root = observer((props: AppRootProps) => {
               <Schedules query={query} />
             </Route>
             <Route path={getRoutesForPage('schedule')} exact>
-              <Schedule query={query} />
+              <Schedule query={query} basicDataLoaded={basicDataLoaded} />
```

Comment:
this seems odd to me.. why not just only conditionally render the root `<Switch>` component if `basicDataLoaded` is true?

Reply:
Indeed, this looks a bit odd. I'll change it so that it's at the top level instead, a few other pages also seem to depend on the data loaded from there.